futurecommander_gui/node/web/application.js

```diff
@@ -0,0 +1,43 @@
+
+/*
+ * Copyright 2019 François CADEILLAN
+ *
+ * This file is part of FutureCommander.
+ *
+ * FutureCommander is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * FutureCommander is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with FutureCommander.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+const m = nw.require('mithril');
```

Comment:
Sometime there's `m` and sometime there's `mithril` Is there a difference ?

I propose to use only one of them for consistency

Reply:
The underlying (unconscious) idea behind that is `mithril` is used a global rendering dependency ( for mount, redraw or just as a forwad dependency ) while `m` is used as a pure view component renderer ( views only ).

In one hand i hate single letters dependency, in other it's PITA to type whole `mithril` word for each HTML element ... 

I don't know, what do you think ?

Reply:
In the official documentation of Mithril, `m` is used everywhere, not Mithril.

About writting the same things all the time, you could use editor snippet or even Mithril Emmet extentions.

So ok ! Stick with it, I think it's fine